service/awsconfig/v4/resource/cloudformation/validate.go

```diff
@@ -44,7 +44,7 @@ func (r *Resource) validateHostPeeringRoutes(cluster v1alpha1.AWSConfig) error {
 	}
 	output, err := r.HostClients.EC2.DescribeRouteTables(input)
 	if err == nil && len(output.RouteTables) == 1 {
-		return microerror.Mask(existingRouteError)
+		return microerror.Maskf(existingRouteError, "route already exists: %s", key.PrivateSubnetCIDR(cluster))
```

Comment:
Here error var is a bit too detailed. I try to encourage to create more generic errors.

Right now it's `existing route: route already exists 'name'`.

I'd change that to `alreadyExistsError`, and then `already exists: route 'name'`.
So generic error + details.

Reply:
Good point, done.